packages/ui/src/Link.tsx

```diff
@@ -80,43 +80,54 @@ export type LinkProps = IconProps & {
  * - Link can be used as a stand-alone component with right chevron icon.
  * - You can change underlying semantics with a component property. Typescript will guard you on providing other properties related to the component type.
  */
-export const Link: FC<LinkProps> = ({
-  component: Component = 'a',
-  kind = 'primary',
-  size = 'normal',
-  bold = false,
-  icon,
-  ariaLabel,
-  iconClassName,
-  href,
-  rel = 'noopener',
-  target = '_self',
-  onClick,
-  className,
-  children,
-}) => {
+export const Link = React.forwardRef<HTMLAnchorElement, LinkProps>(function Link(
```

Comment:
```suggestion
export const Link = React.forwardRef<HTMLElement, LinkProps>(function Link(
```
Then we could pass refs to the button as well. WDYT?

Reply:
@aigoncharov I agree, that would enhance the component, but I can't define proper typing this way, I keep getting type errors here
![image](https://user-images.githubusercontent.com/13898415/114155763-842ab680-992a-11eb-9827-ef51d453d9f9.png)

The only way I could think of to properly type the component is to actually make a separate component LinkButton...
Maybe you could help me...

Reply:
@fantkolja pushed a fix